app/templates/events/event_list.html

```diff
@@ -53,137 +53,147 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
-              <td> <a href="/eventsList/{{event.id}}/view" class="link-primary">{{event.name}}</a></td>
-            {% endif %}
-            <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
-            <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-            <td>{{event.location}}</td>
-            {% if user.isAdmin %}
-              <td>
-                {% if event.isPast %}
-                  {% set isPastEvent = "true" %}
+              <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
+              <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
+              <td>{{event.location}}</td>
+              {% if user.isAdmin %}
+                <td>
+                  {% if event.isPast %}
+                    {% set isPastEvent = "true" %}
+                  {% endif %}
+                  <button class="btn btn-primary" onclick="showEmailModal({{event.id}}, 'Unknown', {{selectedTerm}}, {{isPastEvent}})">
+                    <span class="bi bi-envelope-fill"> Invite</span>
+                  </button>
+                </td>
+              {% else %}
+                <td>
+                {% if event.id in rsvpedEventsID %}
+                  <button type="button" class="btn btn-danger" onclick="removeRsvpForEvent({{event.id}})">Remove RSVP</button>
+                {% elif (not event.isPast) and (event.isRsvpRequired) %}
+                  <button type="button" class="btn btn-primary" onclick="rsvpForEvent({{event.id}})">RSVP</button>
                 {% endif %}
-                <button class="btn btn-primary" onclick="showEmailModal({{event.id}}, 'Unknown', {{selectedTerm}}, {{isPastEvent}})">
-                  <span class="bi bi-envelope-fill"> Invite</span>
-                </button>
-              </td>
-            {% else %}
-              <td>
-              {% if event.id in rsvpedEventsID %}
-                <button type="button" class="btn btn-danger" onclick="removeRsvpForEvent({{event.id}})">Remove RSVP</button>
-              {% elif (not event.isPast) and (event.isRsvpRequired) %}
-                <button type="button" class="btn btn-primary" onclick="rsvpForEvent({{event.id}})">RSVP</button>
+                </td>
               {% endif %}
-              </td>
+            </tr>
+          {% endif %}
+          {% if user.isCeltsStudentStaff or user.isAdmin %}
+            {% if not event.isPast and events[events.index(event)+1].isPast %}
+              <tr>
+                <td>
+                  <button class="toggle-event btn btn-secondary">
+                    <span class="bi "> Past Events</span>
+                  </button>
+                </td>
+              </tr>
             {% endif %}
-          </tr>
-        {% endfor %}
-     </tbody>
-   </table>
- </div>
+            {% if event.isPast %}
+              <tr class=" {{visibleClass}} table-secondary {{visibleEvent}}">
```

Comment:
change `table-secondary` to `table-light`.

Reply:
I thought that past events should be shown by default when there are no upcoming events. It was mentioned in a previous comment by Finn